### PR TITLE
Change opossum sprites | Add new pet poppy the safety opossum

### PR DESCRIPTION
**Pull request by VasilisThePikachu**
## About the PR

Changes the opossum sprites to ones from baystation (it was the only ones i could fine and i cant sprite)

Also adds poppy, a safety opossum! Would go great as an engineering pet. (they dont have a pet right? only atmos does)

Since I know a bunch of people love Morty and his iconic sprite. I kept him in as an admin spawn

Little discussion

https://discord.com/channels/310555209753690112/346743671787356160/1174379481436270592  (deltanedas cant read this :troll:)

## Why / Balance

Old sprites were only 2 directional and were not really in match of the art style. Neither are these really in the right art style but its better then what we have right now

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

Poppy
image
image

Morty/Possums
image
image

## Breaking changes

**Changelog**

:cl: Vasilis
- add: Added Poppy the Safety Opposum!
- tweak: Opossums have received some new sprites. Old sprites are still available to be spawned in by admins.

### CODE REVIEW

**Comment by CatMagic1** [comment]
buh morty be mortying why he no mortying no more

**Comment by UbaserB** [comment]
nooo morty no longer morty :(

**Comment by Emisse** [comment]
morty killed

File: Resources/Textures/Mobs/Animals/possum.rsi/meta.json [hunk 1]
```diff
@@ -1,21 +1,35 @@
 {
   "version": 1,
-  "size":
-  {
+  "license": "CC-BY-SA-3.0",
+  "copyright": "New sprites taken from Baystation12 at https://github.com/Baystation12/Baystation12/blob/c7442716b52ce4edeef5eaf5d2fc7c57a08a3f01/icons/mob/simple_animal/possum.dmi, old sprites taken from goon https://github.com/goonstation/goonstation/commit/c98666f47a90d106c7a154cedffc747b3769e933. CC-BY-NC-SA-3.0",
```
Comment (DrSmugleaf) [inline Resources/Textures/Mobs/Animals/possum.rsi/meta.json:4]: This should be a separate .rsi instead of mixing licenses in the copyright field like this

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Comment by UbaserB** [comment]
NOOOOOOOOOO morty

**Comment by Blackern5000** [comment]
MORTY NOOOOOOOOOOOOOOOOOOOO

**Comment by VasilisThePikachu** [comment]
Oh no ..... I can already see the assassin's coming to my house

**Comment by MACMAN2003** [comment]
it's mortyover

**Pull request merged**